src/components/SearchOverlay/SearchOverlay.js

```diff
@@ -325,112 +198,59 @@ class SearchOverlay extends React.PureComponent {
           searchUp: isSearchUp,
           ambientString: isAmbientString,
         },
-        results,
+        searchResults,
       );
     });
   };
 
   onChange = e => {
-    const { isSearchPanelOpen, setSearchValue } = this.props;
+    const { setSearchValue } = this.props;
     const searchValue = e.target.value;
-
     setSearchValue(searchValue);
-
-    // if (searchValue.trim()) {
-    //   if (isSearchPanelOpen) {
-    //     this.clearSearchResults();
-    //     this.executeDebouncedFullSearch();
-    //   } else {
-    //     this.executeDebouncedSingleSearch();
-    //   }
-    // } else {
-    //   this.clearSearchResults();
-    // }
   };
 
   search = () => {
     const { isSearchPanelOpen } = this.props;
-
     this.searchTextInput.current.blur();
     if (isSearchPanelOpen) {
       this.clearSearchResults();
       this.executeDebouncedFullSearch();
-    } else {
-      this.executeDebouncedSingleSearch();
     }
   }
 
   onKeyDown = e => {
-    const shouldOpenSearchPanel =
-      !this.props.isSearchPanelDisabled &&
-      (e.metaKey || e.ctrlKey) && e.which === 13;
-
-    if (e.shiftKey && e.which === 13) {
-      // Shift + Enter
-      this.onClickPrevious(e);
-    } else if (shouldOpenSearchPanel) {
-      // (Cmd/Ctrl + Enter)
-      this.onClickOverflow(e);
-    } else if (e.which === 13) {
+    if (e.which === 13) {
```

Comment:
just following what was there

Reply:
I see, but I think it doesn't hurt to put that into a variable with a meaningful name. What do you say?

Reply:
I actually prefer in these cases having the actual value and then the meaning on comments. Let's me easily see both rather than having here "Key.Enter" and then I need to go to another file to see what is the actual value.

We also seem to use values in other places, so I would follow that for now and if we want to change this, we should consider making it so that we'll change it our coding styles

Reply:
Aight!